tests/models/test_mistral.py

```diff
@@ -187,13 +188,20 @@ def test_init():
 
 
 async def test_multiple_completions(allow_model_requests: None):
+    from datetime import datetime, timezone
+
     completions = [
+        # First completion: created is "now" (simulate IsNow)
         completion_message(
             MistralAssistantMessage(content='world'),
             usage=MistralUsageInfo(prompt_tokens=1, completion_tokens=1, total_tokens=1),
-            with_created=False,
```

Comment:
The `coverage` check is complaining that lines 328 and 363 in `mistral.py` are no longer covered by tests, because we're no longer passing `created=None` here. Can you restore this? If you get a type error, we can hide it with a `# pyright: ignore` comment at the end of the line.